src/v2/Apps/Artwork/routes.tsx

```diff
@@ -10,10 +11,28 @@ export const routes = [
     prepare: () => {
       ArtworkApp.preload()
     },
+    prepareVariables: ({ artworkID }, props) => {
```

Comment:
Use `prepareVariables` to control including the artist series fragment or not. Eventually when this gets rolled out to everyone we could just remove all the `@include` stuff.

Reply:
We end up doing this `userHasLabFeature` check in two different places — is that to account for SSR vs CSR?

Reply:
What do you mean, here _and_ in the component? (This is all server side rendered btw).

It's more that it's two separate uses. This one is for a GraphQL variable to make sure MP isn't doing wasted work by loading artist series stuff. Then, the component guard is to not render the rail (where the data wouldn't have been fetched anyway, by virtue of the corresponding fragment being left out).

Reply:
Ah right, makes sense.